peribolos/knative-extensions.yaml

```diff
@@ -16,7 +16,6 @@ orgs:
     - matzew
     - nrrso
     # cncf
-    - thelinuxfoundation
```

Comment:
I think we're supposed to keep this one.

Reply:
they never accepted the invite so unsure what they're supposed to do

Reply:
added it back